tests/cross_functional/performance/csi_tests/test_pvc_multi_snapshot_performance.py

```diff
@@ -176,7 +176,10 @@ def teardown(self):
 
             # Deleting the Data pool
             log.info(f"Deleting the test storage pool : {self.sc_name}")
-            self.delete_ceph_pool(self.sc_name)
+            try:
+                self.delete_ceph_pool(self.sc_name)
+            except Exception:
```

Comment:
This is not a good idea to pass on any exception. what error are you trying to handle here?

Reply:
@keesturam - this is the exception I'm trying to handle : 

failed on teardown with "ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: oc -n openshift-storage patch cephblockpool pas-testing-rbd --type=merge -p '{"metadata":{"finalizers":null}}'.
Error is Error from server (NotFound): cephblockpools.ceph.rook.io "pas-testing-rbd" not found"

from https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-test-pr/6605/testReport/tests.cross_functional.performance.csi_tests.test_pvc_multi_snapshot_performance/TestPvcMultiSnapshotPerformance/test_pvc_multiple_snapshot_performance_CephBlockPool_512_/

Reply:
This is an occasional exception during Teardown of the test, it does not happen on every run. 
We handle similar issue the same way in the test_pvc_clone_performance.py test ( lines 143-145) .